src/components/Components.css

```diff
@@ -67,9 +67,20 @@ ion-modal {
   padding-left: 0.5rem;
 }
 
-/* 
+/* something about icons being multiples of 8 pixels in width*/
+#prevButton {
+  --ionicon-stroke-width: 32px;
+  font-size: 72px;
+}
+
+#nextButton {
```

Comment:
Since these are the same it could prob be just 1 css thing and use it for both buttons